src/Database/Database.php

```
@@ -5019,16 +5053,23 @@ public function updateDocuments(
         );
 
-        // Check new document structure
-        $validator = new PartialStructure(
-            $collection,
-            $this->adapter->getIdAttributeType(),
-            $this->adapter->getMinDateTime(),
-            $this->adapter->getMaxDateTime(),
-        );
+        // Separate operators from regular updates for validation
+        $extracted = Operator::extractOperators($updates->getArrayCopy());
+        $operators = $extracted['operators'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$operators'. (undefined)

(UnusedLocalVariable)